src/test/integration/mocks/notification-message.ts

```diff
@@ -38,9 +38,14 @@ export const assertInfoMessageDisplayed = async (message: string) => {
 };
 
 export const assertErrorMessageDisplayed = async (message: string) => {
-  // wait for setInterval to invoke Environments provider state refresh
-  await waitFor(() => expect(showErrorMessageSpy).toHaveBeenCalled());
-  expect(showErrorMessageSpy).toHaveBeenCalledWith(message, "More info");
+  await waitFor(() => {
```

Comment:
isn't it problematic to have other stuff call that function when we didn't mean to?

Reply:
f934e53f992969803ee62d494ed42b0be6b00b0d
124a2146d61f1add26d4c3d7c8eef629895ca274

Reply:
extract error messages to errors file
added ansi-strip import to tests since we import errors file into the tests